imports/api/Constants.js

```diff
@@ -13,11 +13,23 @@ export const Constants = {
 	// from noten players to tenpai ones in a Japanese game
 	JPN_TENPAI_PAYOUT: 3000,
 
-	// The required number of points of any one player to 
+	// The required number of points of any one player to
 	// end a game at the end of "South" round in a Japanese
 	// game
 	JPN_END_POINTS: 30000,
 
+	// Base points of a mangan
+	JPN_MANGAN_BASE_POINTS: 2000,
+
+	// Points for every bonus round
+	BONUS_POINTS: 300,
+
+	// Points for every riichi stick
+	RIICHI_POINTS: 1000,
+
+	// Points paid for a mistake
+	MISTAKE_POINTS: 12000,
```

Comment:
Might as well give the JPN_ prefix to bonus and Riichi too, since that is sort of the convention it seems.